src/main/index.ts

```diff
@@ -1,8 +1,14 @@
 // Main process
 
 import "../common/system-ca"
-import "../common/prometheus-providers"
 import { app, dialog, protocol } from "electron"
+if (process.env.NODE_ENV === "development") {
```

Comment:
I would use `isDevelopment` constant from `common/vars.ts` and probably put both constants definition near each other (for better readability), so:
```
  const appName = 'LensDev';
  const appData = app.getPath('appData');
  app.setName(appName);
  app.setPath('userData', path.join(appData, appName));
```

Reply:
I agree that it would be more readable but `setName` needs to happen before `getPath`. Changed to use `isDevelopment`.